test/cmd/status.sh

```diff
@@ -21,38 +21,38 @@ cp "${KUBECONFIG}" "${login_kubeconfig}"
 
 os::test::junit::declare_suite_start "cmd/status"
 # login and ensure no current projects exist
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/server-ca.crt' -u test-user -p anything"
+os::cmd::expect_success_and_text "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/server-ca.crt' -u test-user2 -p anything" 'You can try to create'
```

Comment:
Why you're changing tests?

Reply:
Debugging. I'll change it back. Need to figure out why the commands in this test behave differently when run via test-cmd.sh